package.json

```diff
@@ -8,36 +8,36 @@
   },
   "scripts": {
     "build:dev": "webpack",
-    "build:prod": "NODE_ENV=production webpack -p"
+    "build:watch": "webpack --watch",
+    "build:prod": "webpack -p --env production"
   },
   "dependencies": {
-    "@babel/preset-react": "^7.0.0",
-    "babel-core": "^6.26.3",
-    "babel-loader": "^7.1.5",
-    "babel-plugin-transform-react-constant-elements": "6.23.0",
-    "babel-plugin-transform-react-inline-elements": "6.22.0",
-    "babel-plugin-transform-react-jsx": "6.24.1",
-    "babel-preset-es2015": "6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-2": "6.24.1",
     "bootstrap": "3.4.1",
+    "create-react-class": "^15.6.3",
```

Comment:
this is duplicated in devDependencies, and is now no longer needed anyway because we removed all instances of createClass